README.md

```diff
@@ -27,64 +27,28 @@ Also see [buildah-build](https://github.com/redhat-actions/buildah-build) for mo
 
 Once an image has been built, [push-to-registry](https://github.com/redhat-actions/push-to-registry) can be used to push it to an image registry.
 
+<a id="action-inputs"></a>
+
 ## Action Inputs
 
-<table>
-  <thead>
-    <tr>
-      <th>Input</th>
-      <th>Required</th>
-      <th>Default</th>
-      <th>Description</th>
-    </tr>
-  </thead>
-
-  <tr>
-    <td>builder_image</td>
-    <td>Yes</td>
-    <td>-</td>
-    <td>
-      The location of the S2I builder image. A curated list of builder images can be found
-      <a href="./builder-images.md">here</a>.
-    </td>
-  </tr>
-
-  <tr>
-    <td>image_name</td>
-     <td>Yes</td>
-    <td>-</td>
-    <td>The name of the image to produce. </td>
-  </tr>
-
-  <tr>
-    <td>image_tag</td>
-    <td>No</td>
-    <td>latest</td>
-    <td>The tag of the image to produce.</td>
-  </tr>
-
-  <tr>
-    <td>path_context</td>
-    <td>No</td>
-    <td><code>.</code></td>
-    <td>The location of the path to run S2I from. This should be the path where your source code is stored.</td>
-  </tr>
-
-  <tr>
-    <td>log_level</td>
-    <td>No</td>
-    <td>1</td>
-    <td><a href="https://github.com/openshift/source-to-image/blob/master/docs/cli.md#log-levels">Log level</a> when running S2I. Can be 0 (least verbose) to 5 (most verbose).</td>
-  </tr>
-
-  <tr>
-    <td>env_vars</td>
-    <td>No</td>
-    <td>-</td>
-    <td>List of environment variable key-value pairs to pass to the s2i builder context. (eg. <code>key=value</code>, <code>mysecret=${{ secrets.MY_SECRET }}</code> ).</td>
-  </tr>
-
-</table>
+| Input Name | Description | Default |
+| ---------- | ----------- | ------- |
+| builder_image | The location of the S2I builder image. A curated list of builder images can be found [here](./builder-images.md). | **Required**
+| env_vars | List of environment variable key-value pairs to pass to the S2I builder context. (eg. `key=value`, `mysecret=${{ secrets.MY_SECRET }}`). | None
+| image_name | Name to give to the output image. | **Required**
+| image_tags | The tags of the image to build. For multiple tags, separate by a space. For example, `latest ${{ github.sha }}` | `latest`
```

Comment:
I think just `tags` is better to match buildah/ptr

Reply:
oh, i didn't realize we already had `image_tag`.

Reply:
up to you if we change it or not since we're doing a major version anyway

Reply:
Okay, I will change this ta match other actions.